Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,28 @@
+require "language/go"
```

Comment:
Homebrew doesn't use binaries. ` go ` is only a build-time dep, so if people just do ` brew install vegeta ` they won't need ` go ` installed.